README.md

```diff
@@ -49,7 +49,7 @@ MSG_RETRY_THRESHOLD=3
 ```
 
 # Running
-If you went through the previous steps successfully, you should now be able to run **Pepe as a Service**™ with the following command:
+If you went through the previous steps successfully you should now be able to run **Pepe as a Service**™ with the following command:
```

Comment:
Try swapping the sentence from the left of the comma with the one from the right and see if it still makes sense.

Original:
`If you went through the previous steps successfully, you should now be able to run **Pepe as a Service**™ with the following command.`

Swapped:
`You should now be able to run **Pepe as a Service**™ with the following command, if you went through the previous steps successfully.`

Make sense? Leave it, then.